### ISSUE TITLE
Accessibility - Report section collapse expand button not read out to screen readers

### ISSUE THREAD

**Issue by aj-he** [archesproject/arches-her#1289]
**Given** I am a user reliant on a screen reader
**When** I navigate through the headings on the resource reports
**Then** The screen reader announces the expand/collapse box only as "clickable" or "expand/collapse" rather than if it is expanded or collapsed

This could leave the user unsure if the heading is accessible of they have accidentally collapsed it, meaning they are unable to access its content

**4.1.2 – Name, Role, Value**
Assistive Technologies (AT) should be able to gather information about, activate(or set) and keep up to date on the status of user interface controls in the content. If custom controls are created, however, or interface elements are programmed (in code or script) to have a different role and/or function than usual, then additional measures need to be taken to ensure that the controls provide important information to assistive technologies and allow themselves to be controlled by assistive technologies. Examples include
- A particularly important state of a user interface control is whether or not it has focus. The focus state of a control can be programmatically determined, and notifications about change of focus are sent to user agents and assistive technology.
 - Other examples of user interface control state are whether or not a checkbox or radio button has been selected, or whether or not a collapsible tree or list node is expanded or collapsed.

**Comment by aj-he** [comment]
closed complete in #1301
